products/ubuntu2004/profiles/cis_level1_server.profile

```diff
@@ -880,19 +888,19 @@ selections:
     - no_rsh_trust_files
 
     ### 6.2.12 Ensure all groups in /etc/passwd exist in /etc/group (Automated)
-    # Needs rule: all_etc_passwd_groups_exist_in_etc_group
+    - gid_passwd_group_same
 
     ### 6.2.13 Ensure no duplicate UIDs exist (Automated)
-    # Needs rule: no_duplicate_uids
+    - account_unique_id
 
     ### 6.2.14 Ensure no duplicate GIDs exist (Automated)
-    # Needs rule: no_duplicate_gids
+    - group_unique_id
 
     ### 6.2.15 Ensure no duplicate user names exist (Automated)
-    # Needs rule: no_duplicate_user_names
+    - account_unique_name
 
     ### 6.2.16 Ensure no duplicate group names exist (Automated)
-    # Needs rule: no_duplicate_group_names
+    - group_unique_name
 
     ### 6.2.17 Ensure shadow group is empty (Automated)
-    # Needs rule: ensure_shadow_group_empty
+    - ensure_shadow_group_empty
```

Comment:
Could you include an empty line at the end, please?

Reply:
done